eppo/src/androidTest/java/cloud/eppo/android/EppoClientTest.java

```diff
@@ -51,15 +50,15 @@
 import cloud.eppo.android.dto.FlagConfig;
 import cloud.eppo.android.dto.RandomizationConfigResponse;
 import cloud.eppo.android.dto.SubjectAttributes;
-import cloud.eppo.android.dto.deserializers.EppoValueAdapter;
+import cloud.eppo.android.dto.adapters.EppoValueAdapter;
```

Comment:
Now that I understand more about Adapters, I renamed this package back to `adapters` (how it was earlier)